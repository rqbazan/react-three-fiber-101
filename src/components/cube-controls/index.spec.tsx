import React from 'react'
import { render } from '@testing-library/react'
import CubeControls, { CubeControlsProps } from './index'

function renderCubeControls(props?: Partial<CubeControlsProps>) {
  const defaultProps: CubeControlsProps = {
    onControlClick: () => {},
    preferLetters: true
  }
  return render(<CubeControls {...defaultProps} {...props} />)
}

it.skip('updates controls when a slice controls in clicked', async () => {
  expect(1).toBe(1)
})
