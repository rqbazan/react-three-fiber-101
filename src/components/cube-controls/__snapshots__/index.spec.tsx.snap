// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`updates controls when a slice controls in clicked 1`] = `
<div>
  <div
    class="container left"
  >
    <button
      class="button rounded-full border-solid border-3 border-black h-10 w-10 p-0 bg-pegatine-green"
      data-testid="front"
      type="button"
    >
      F'
    </button>
    <button
      class="button rounded-full border-solid border-3 border-black h-10 w-10 p-0 bg-pegatine-white"
      data-testid="down"
      type="button"
    >
      L'
    </button>
    <button
      class="button rounded-full border-solid border-3 border-black h-10 w-10 p-0 bg-pegatine-orange"
      data-testid="right"
      type="button"
    >
      D'
    </button>
    <button
      class="button rounded-full border-solid border-3 border-black h-10 w-10 p-0 bg-pegatine-blue"
      data-testid="back"
      type="button"
    >
      B'
    </button>
    <button
      class="button rounded-full border-solid border-3 border-black h-10 w-10 p-0 bg-pegatine-yellow"
      data-testid="up"
      type="button"
    >
      R'
    </button>
    <button
      class="button rounded-full border-solid border-3 border-black h-10 w-10 p-0 bg-pegatine-red"
      data-testid="left"
      type="button"
    >
      U'
    </button>
    <button
      class="button rounded-full border-solid border-3 border-black h-10 w-10 p-0"
      data-testid="middle"
      type="button"
    >
      M'
    </button>
    <button
      class="button rounded-full border-solid border-3 border-black h-10 w-10 p-0"
      data-testid="standing"
      type="button"
    >
      S'
    </button>
    <button
      class="button rounded-full border-solid border-3 border-black h-10 w-10 p-0"
      data-testid="equatorial"
      type="button"
    >
      E'
    </button>
  </div>
  <div
    class="container right"
  >
    <button
      class="button rounded-full border-solid border-3 border-black h-10 w-10 p-0 bg-pegatine-green"
      data-testid="front"
      type="button"
    >
      F
    </button>
    <button
      class="button rounded-full border-solid border-3 border-black h-10 w-10 p-0 bg-pegatine-white"
      data-testid="down"
      type="button"
    >
      L
    </button>
    <button
      class="button rounded-full border-solid border-3 border-black h-10 w-10 p-0 bg-pegatine-orange"
      data-testid="right"
      type="button"
    >
      D
    </button>
    <button
      class="button rounded-full border-solid border-3 border-black h-10 w-10 p-0 bg-pegatine-blue"
      data-testid="back"
      type="button"
    >
      B
    </button>
    <button
      class="button rounded-full border-solid border-3 border-black h-10 w-10 p-0 bg-pegatine-yellow"
      data-testid="up"
      type="button"
    >
      R
    </button>
    <button
      class="button rounded-full border-solid border-3 border-black h-10 w-10 p-0 bg-pegatine-red"
      data-testid="left"
      type="button"
    >
      U
    </button>
    <button
      class="button rounded-full border-solid border-3 border-black h-10 w-10 p-0"
      data-testid="middle"
      type="button"
    >
      M
    </button>
    <button
      class="button rounded-full border-solid border-3 border-black h-10 w-10 p-0"
      data-testid="standing"
      type="button"
    >
      S
    </button>
    <button
      class="button rounded-full border-solid border-3 border-black h-10 w-10 p-0"
      data-testid="equatorial"
      type="button"
    >
      E
    </button>
  </div>
</div>
`;
